app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -159,4 +159,38 @@ public function processInvitationcall($observer)
         }
     }
 
+    /**
+     * @throws Exception
+     */
+    public function sendSyncUrl(Varien_Event_Observer $observer) {
+        if (
+            Mage::getStoreConfig('webwinkelconnect/product_review_invites/enabled')
+            && Mage::getStoreConfig('webwinkelconnect/general/enabled')
+        ) {
+            $helper = Mage::helper('webwinkelconnect');
+            try {
```

Comment:
A try/catch should be around the part that is actually expected to fail, not around the "setup code".

It looks like the Magento cURL adapter doesn't even throw exceptions, so I wonder which exceptions you're trying to catch. Can you clarify that?

Reply:
I was trying to catch cURL exceptions, but upon further check after your comment I have removed the try catch statement and have implemented a different way of dealing with the cURL errors.